src/dash/models/DashManifestModel.js

```diff
@@ -41,6 +41,12 @@ import ObjectUtils from '../../streaming/utils/ObjectUtils';
 import URLUtils from '../../streaming/utils/URLUtils';
 import FactoryMaker from '../../core/FactoryMaker';
 
+Number.isInteger = Number.isInteger || function (value) {
```

Comment:
I would prefer to see a local function within DashManifestModel to avoid modifying `Number` within dash.js.

```
function DashManifestModel(config) {
    const isInteger = Number.isInteger || function (value) { ...
...
    if (realAdaptations && isInteger(index)) { ...
```